front/app/containers/Admin/projects/project/nativeSurvey/utils.test.ts

```diff
@@ -41,247 +41,9 @@ describe('resetOptionsIfNotPersisted', () => {
     },
   ];
 
-  it('should not alter the fields if the form is already persisted', () => {
-    const resultCustomFields = resetOptionsIfNotPersisted(customFields, true);
-    expect(resultCustomFields).toEqual(customFields);
-  });
-
-  it('should remove the ID from options and add a tempID', () => {
-    const resultCustomFields = resetOptionsIfNotPersisted(customFields, false);
-    expect(resultCustomFields[0].options[0].id).toBe(undefined);
-    expect(resultCustomFields[0].options[1].id).toBe(undefined);
-    expect(resultCustomFields[0].options[0].temp_id).toMatch(/^TEMP-ID-/);
-    expect(resultCustomFields[0].options[1].temp_id).toMatch(/^TEMP-ID-/);
-  });
-});
-
-describe('resetCopiedForm', () => {
-  const customFields: IFlatCustomField[] = [
```

Comment:
Just want to make sure we did indeed want to remove these tests?

Reply:
Yes, all the logic for duplicating the form (apart from removing the option IDs) is now in the backend